src/Controllers/BatchProcessorController.ts

```diff
@@ -48,6 +48,9 @@ module StreamStats.Controllers {
         public AppVersion: string;  
         public submitBatchInfo: string;
         public regionList: Object;
+        public flowStatsList: Object;
+        public selectedRegion: any;

```

Comment:
```suggestion
        public selectedRegion: str;

```

We want to avoid "any" went possible